test/passing/gen/gen.ml

```diff
@@ -7,7 +7,8 @@ type setup =
   ; mutable base_file: string option
   ; mutable extra_deps: string list
   ; mutable should_fail: bool
-  ; mutable minimum_ocaml_version: string option }
+  ; mutable minimum_ocaml_version: string option
+  ; mutable enabled_if: string option }
```

Comment:
@gpetiot I had to add a more general mechanism of disabling tests. We can either:

- keep both `.minimum-ocaml-version` and `.enabled-if` files, where the former is a shortcut for a version check and there's an error if both files are provided
- just keep `.enabled-if` files and migrate the existing `.minimum-ocaml-version` files to this new format

What do you prefer?

Reply:
I would prefer to have only one test, so only the `.enabled-if`

Reply:
done!